README.md

```diff
@@ -56,7 +56,7 @@ Version 4 (random):
 
 ```javascript
 const uuidv4 = require('uuid/v4');
-uuidv4(); // ⇨ '3a017fc5-4f50-4db9-b0ce-4547ba0a1bfd'
+uuidv4(); // ⇨ 'e57d7cee-a174-4463-9f89-1ada39083c5f'
```

Comment:
Why did this value change?

Reply:
See <https://github.com/kelektiv/node-uuid/pull/293#issuecomment-425570962>

Reply:
See the [runmd](https://github.com/broofa/runmd) module.  tl;dr: 'Got fed up with broken README code samples.  'Wrote a utility to evaluate/annotate said examples.  Said utility randomly generates UUIDs each time it's run because random is as random does.